deploy/canary/Dockerfile

```diff
@@ -1,10 +0,0 @@
-FROM golang:latest
```

Comment:
Why delete this file? This Dockerfile makes it possible to build images using just `docker build ...`?
Once we get the jenkins automatic builder to work reliably using makefiles, I'd be comfortable with removing this Dockerfile.